solana-programs/reward-manager/program/src/processor.rs

```diff
@@ -81,6 +85,25 @@ impl Processor {
         Ok(())
     }
 
+    fn process_change_reward_manager_authority<'a>(
+        reward_manager_info: &AccountInfo<'a>,
+        current_authority_info: &AccountInfo<'a>,
+        new_authority_info: &AccountInfo<'a>,
+    ) -> ProgramResult {
+        if !current_authority_info.is_signer {
+            return Err(ProgramError::MissingRequiredSignature);
+        }
+
+        let mut reward_manager = RewardManager::unpack(&reward_manager_info.data.borrow())?;
+        assert_account_key(current_authority_info, &reward_manager.manager)?;
+
+        reward_manager.manager = *new_authority_info.key;
```

Comment:
is there a way to set this to the equivalent of a zero address? in order to effectively 'disable' it

Reply:
Yes, you can set it to zero address